docs/content/reference/resource-schema/link-schema/dapr-schema/dapr-pubsub/snippets/dapr-pubsub-manual.bicep

```
@@ -34,7 +38,10 @@ resource pubsub 'Applications.Link/daprPubSubBrokers@2022-03-15-privatepreview'
   properties: {
     environment: environment
     application: app.id
-    mode: 'values'
+    resourceProvisioning: 'manual'
+    resources: [
+      { id: account.id }
```

```suggestion
      { id: kafkaRoute.id }
```

Usually the user would specify the resource they're referencing and using for this instead of a different resource